_utils/install_all_solutions.py

```diff
@@ -44,10 +42,11 @@ def install_all_solutions():
     if solution.strip() == "all":
         install_all_solutions()
     else:
-        if solution == "clickhouse":
-            continue
-        elif solution == "data.table":
+        if solution == "data.table":
             install_solution("datatable")
+        elif solution == "clickhouse":
+            install_solution("clickhouse")
+            install_solution("polars")
```

Comment:
Why polars in needed? Just a typo?

Reply:
polars was needed because clickhouse outputs NA as a result in the `time.csv` file. The current verification script throws these out before checking that all results are the same. If only clickhouse is run, then the results are empty after NA is thrown out. So if I add polars then there are still query results that can be verified. Polars was chosen because it installs quickly and runs quickly. 

Feel free to modify how clickhouse writes to `time.csv` to fix this. It doesn't have to be in this PR though, can be later

Reply:
clickhouse/clickhouse-parse-log.R

This is the place to fix it